tests/functional/test_hic.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+
+import pytest
+
+
+@pytest.mark.workflow(name="test_hic")
+def test_hic_alignable_bam_match(workflow_dir, bam_md5):
+    bam_path = workflow_dir / Path("test-output/result_alignable_dedup.bam")
+    bam_md5sum = bam_md5(bam_path)
+    assert bam_md5sum == "0bc6522586908f128d0f24a4dafe4065"
```

Comment:
If the bams you compare to are not huge (or if they are needed for something and would be in the repo anyway), you could also do the comparison the way we do it in dnase (I am almost sure you have your reasons to do it this way, just saying just in case..).